app/controllers/application_controller.rb

```diff
@@ -109,4 +109,16 @@ def set_current_context_for_logging!
   rescue => e
     Rails.logger.error(e)
   end
+
+  def sanitize_order_by(model, order_by, default = nil)
```

Comment:
how many of our controllers would ever use this? I'm not sure if this is the best way to share code for a minority of subclasses

Reply:
Moved it to a concern.